src/utilities/navigateClause.js

```diff
@@ -1,6 +1,7 @@
 import * as R from 'ramda';
 
 import store from '../store';
+import { min } from 'moment-mini';
```

Comment:
do we need this extra library? doesn't look like you are actually using it.

Reply:
I think we are here: https://github.com/accordproject/template-studio-v2/pull/104/files#diff-16f57a2341a65f078824cec5da6ef4e8R70

@archhhh why is this library used?

Reply:
Yes, no need to use it, I removed that library in the last commit.